### DISCUSSION TITLE
Tekton PipelineRun Works Directly But Not Using the Kubernetes-Client

### DISCUSSION THREAD

**Discussion by kjq** [tektoncd/operator#1583] [Q&A] [answered]
Cross-posting to try and get an answer/resolution (https://github.com/fabric8io/kubernetes-client/discussions/5261) because we are stumped...

We have two pipelines, one using a PVC and another not. Using the latest Tekton operator (v.0.67.0), both work in minikube or when applied maually/directly to the EKS cluster using the PipelineRun.

When we go to trigger the pipeline using a PVC using the Quarkus Kubernetes-Tekton Client we get this error when a PVC is involved:

Failure executing: POST at: [https://172.20.0.1:443/apis/tekton.dev/v1beta1/namespaces/pipeline/pipelineruns](https://172.20.0.1/apis/tekton.dev/v1beta1/namespaces/pipeline/pipelineruns). Message: admission webhook \"webhook.pipeline.tekton.dev\" denied the request: mutation failed: cannot decode incoming new object: json: unknown field \"claims\". Received status: Status(apiVersion=v1, code=400, details=null, kind=Status, message=admission webhook \"webhook.pipeline.tekton.dev\" denied the request: mutation failed: cannot decode incoming new object: json: unknown field \"claims\", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=BadRequest, status=Failure, additionalProperties={}).

Is there anything we are missing? We are not running this in OpenShift.

We saw this ticket which seems similar: https://github.com/openshift-pipelines/pipelines-as-code/issues/750

**Comment by jkandasa** [comment]
@kjq What is the `tkn` client version? Can you share the pipeline to reproduce the issue?

**Reply by kjq** [reply]
Sure I can sure our pipeline but it is a collection of tasks and such so I do not know what value it provides.  As mentioned, it works perfectly fine when apply directly to Tekton in the EKS cluster just not when the same PipelineRun is created using the Fabric 8 Kubernetes Client + Tekton.  As for versions, we are using the Tekton Operator installed via Helm 0.67 and everything that comes along with it.

**Comment by kjq** [comment, marked as answer]
We _think_ we worked it out:

https://groups.google.com/g/quarkus-dev/c/VCM0zxVYkJI
